### ISSUE TITLE
Why there are two custom properties (--text-primary-color and --primary-text-color) with such similar names?

### ISSUE THREAD

**Issue by abdonrd** [PolymerElements/polymer-starter-kit#436]
In the [app/styles/app-theme.html](https://github.com/PolymerElements/polymer-starter-kit/blob/master/app/styles/app-theme.html):

```css
  /* Application theme */

  :root {
    --dark-primary-color: #303F9F;
    --default-primary-color: #3F51B5;
    --light-primary-color: #C5CAE9;
    --text-primary-color: #ffffff; /*text/icons*/
    --accent-color: #FF4081;
    --primary-background-color: #c5cae9;
    --primary-text-color: #212121;
    --secondary-text-color: #727272;
    --disabled-text-color: #bdbdbd;
    --divider-color: #B6B6B6;
```

Why there are two custom properties (`--text-primary-color` and `--primary-text-color`) with such similar names?

**Comment by addyosmani** [comment]
This was originally based on https://github.com/PolymerElements/paper-styles/blob/master/default-theme.html. Looking at the spec from the Material Design palette, I do wonder if it's possible for us to make the naming here more clear. @morethanreal or @notwaldorf would either of you have suggestions on how we could improve the variable names used in default-theme to be less confusing?

<img width="849" alt="screen shot 2015-10-13 at 14 41 51" src="https://cloud.githubusercontent.com/assets/110953/10456297/93810fc0-71b8-11e5-8348-f033efaf4bc6.png">

**Comment by reiaguilera** [comment]
:+1:

**Comment by addyosmani** [comment]
Friendly ping @notwaldorf in case you get a chance to check in on this :) I know you've got your hands full!

**Comment by abdonrd** [comment]
Friendly ping to @addyosmani and @notwaldorf. :)

**Comment by notwaldorf** [comment]
Here's the answer I left in https://github.com/PolymerElements/paper-styles/issues/81:
 
I honestly don't know the origin of this, and unfortunately we're now in a situation where both are probably in use, so none can be deleted without being seen as a giant breaking change. :(

`primary-text-color` looks like the right one (87% of #000), and `text-primary-color` is white and I don't know what it's intentions are. I will add a comment to try to discourage people from using it. 

I know this isn't a very satisfying answer, but it's what I got!

**Comment by addyosmani** [comment]
Thanks for the answer! I think discouraging use of `text-primary-color` makes sense. Should we exclude it from PSK and MaterialPalette.com's PSK theme files moving forward? Given I don't know either what the intent there was it seems (eep) harmless to remove.

**Comment by abdonrd** [comment]
Thanks both!

@notwaldorf the https://github.com/PolymerElements/paper-styles/pull/82 is 👌

@addyosmani I think that we should to migrate the PSK and MaterialPalette.com's PSK theme files to the new default-theme.html. @notwaldorf created here: https://github.com/PolymerElements/paper-styles/pull/82

**Comment by addyosmani** [comment]
Sgtm

**Comment by abdonrd** [comment]
@ebidel now you can check here: https://github.com/PolymerElements/polymer-starter-kit/issues/612

**Comment by abdonrd** [comment]
I think that we can close this PR. WDYT?

**Comment by robdodson** [comment]
yep, fixed in #612
